balancer/internal/wrr/wrr_test.go

```diff
@@ -97,3 +98,8 @@ func testWRRNext(t *testing.T, newWRR func() WRR) {
 func TestRandomWRRNext(t *testing.T) {
 	testWRRNext(t, NewRandom)
 }
+
+func init() {
+	r := rand.New(rand.NewSource(0))
+	grpcrandInt63n = r.Int63n
```

Comment:
This source is not threadsafe - is that OK?

Reply:
It's OK here, they are single thread unit tests.